pyglet_tutorial/main.py

```diff
@@ -0,0 +1,71 @@
+import os
+import pyglet
+
+from pyglet.sprite import Sprite
+from pyglet.gl import glClearColor
+from pyglet.window import (
+    Window,
+    # key
+)
+
+IMAGE_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'images')
+
+
+class Character():
+    def __init__(self):
+        self.x = 400
```

Comment:
If these are tutorials, I'd consider commenting almost every line. It is easy to assume what something is is obvious, but I think in the most basic examples, it doesn't hurt to go overkill. That said, not needed for a first merge.